src/lib/logic/typing/globals/interfaces.ts

```diff
@@ -1,6 +1,13 @@
+import type { GeneralInputProps } from "./proptypes.js";
+
 export interface SelectOption {
   key?: string;
   label: string;
   value: string;
   disabled?: boolean;
 }
+
+export interface FieldInputForm<TName extends string, TExtraTypes extends string = "email"> {
+  readonly name: TName;
+  readonly type: GeneralInputProps["type"] | TExtraTypes;
+}
```

Comment:
🚫 **[eslint]** <[eol-last](https://eslint.org/docs/latest/rules/eol-last)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline required at end of file but not found.
```suggestion
}

```